### PR TITLE
[16.0][ADD] helpdesk_ticket_close_inactive

### PR DESCRIPTION
**Pull request by miquelalzanillas**
cc https://github.com/APSL 162752

@javierobcn @mpascuall @BernatObrador @ppyczko @peluko00 

Adds an option from helpdesk team settings to configure a cron job that automatically closes inactive helpdesk tickets after a specified number of days, set in the helpdesk team settings. 

Also sends notification emails to the customer (partner) and assigned team members, providing a warning after an initial period of inactivity and a final closure notification when the maximum inactivity period is reached.

image

### CODE REVIEW

**Comment by BernatObrador** [comment]
@miquelalzanillas Seems like the runboat has not been created property, can you push again and see if solves it?

**Comment by peluko00** [comment]
It's seems runboat failed in all repos today

File: helpdesk_ticket_close_inactive/i18n/es_ok.po
Comment (peluko00) [inline helpdesk_ticket_close_inactive/i18n/es_ok.po:1]: Why this translation file?

File: helpdesk_ticket_close_inactive/readme/CONFIGURE.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+
+To configure this module, you need to:
+
+- Go to Helpdesk > Settings > Teams.
+- Select a team.
+- Enable 'Auto Close Inactive Ticket' option.
```
Comment (peluko00) [inline helpdesk_ticket_close_inactive/readme/CONFIGURE.md]: Enable 'Automatic closure of inactive tickets' option.

Comment (miquelalzanillas) [reply helpdesk_ticket_close_inactive/readme/CONFIGURE.md]: Fixed. Thanks!

**Review by peluko00 — APPROVED** [review]
LGTM, code review

**Review by mpascuall — APPROVED** [review]
LGTM, tested in runboat

**Review by ppyczko — APPROVED** [review]
LGTM

**Review by BernatObrador — COMMENTED** [review]
LGTM

**Comment by flotho** [comment]
any chance to be merged @OCA/helpdesk-maintainers ?

**Comment by miquelalzanillas** [comment]
Hi @pedrobaeza ,

We've been using this module in production for the last weeks and is working ok.

Can you please invoke our dearest bot? :)

**Pull request merged**